explain-overview.md

```diff
@@ -6,6 +6,10 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
 
 # `EXPLAIN` Overview
 
+> **Note:**
+>
+> If you use the `mysql` client to connect to TiDB you may want to use `pager less -S` to avoid line wrapping in your terminal and instead press the right arrow on your keyboard to horizontally scroll through the explain output.
```

Comment:
+ Does "the right arrow on your keyboard" mean <kbd>></kbd>?
+ `pager less -S` and "the right arrow on your keyboard" are used together?

Reply:
This is how this works:

```
mysql> pager less -S
mysql> EXPLAIN ANALYZE SELECT.....
```
The output is shown in `less` and instead of wrapping lines it chops off the part that doesn't fit on the screen. Then by pressing the right arrow key on the keyboard you can 'scroll' to the right. Here `\G` in MySQL doesn't help.

Example w/o this:
![image](https://user-images.githubusercontent.com/1272980/114981587-8642c700-9e8e-11eb-81fe-7c003b2e368f.png)

Example with less -S:
![image](https://user-images.githubusercontent.com/1272980/114981660-a07ca500-9e8e-11eb-846a-7a50c72dadd4.png)

Here you can see that `less` indicates there is more output on the right with `>`. The output can be viewed by pressing the right arrow key on the keyboard.

This improves the output of the tree-like explain output that is in the `id` column of the explain output.